charts/kube-prometheus-stack/templates/alertmanager/alertmanager.yaml

```diff
@@ -162,4 +162,7 @@ spec:
 {{- if .Values.alertmanager.alertmanagerSpec.forceEnableClusterMode }}
   forceEnableClusterMode: {{ .Values.alertmanager.alertmanagerSpec.forceEnableClusterMode }}
 {{- end }}
+{{- if .Values.alertmanager.alertmanagerSpec.minReadySeconds }}
```

Comment:
This condition will remove the attribute by default (`0`) as to not break STS spec in k8s < 1.23.